automations/data/labels.yml

```diff
@@ -15,6 +15,7 @@ colors:
 groups:
   - name: priority
     is_required: true
+    apply_all_available: false
```

Comment:
Since `false` is so prevalent here, maybe we should make it a default and allow it to be overwritten only when it's `true`? Similar to `is_required`.